plugins/obs-outputs/rtmp-stream.c

```diff
@@ -922,33 +961,41 @@ static inline bool send_headers(struct rtmp_stream *stream)
 
 	if (!send_audio_header(stream, i++, &next))
 		return false;
-
-	// send metadata only if HDR
-	video_t *video = obs_get_video();
-	const struct video_output_info *info = video_output_get_info(video);
-	enum video_colorspace colorspace = info->colorspace;
-	if (colorspace == VIDEO_CS_2100_PQ || colorspace == VIDEO_CS_2100_HLG)
-		if (!send_video_metadata(stream)) // Y2023 spec
-			return false;
-
-	if (!send_video_header(stream))
+	if (!send_video_metadata(stream, 0) || !send_video_header(stream, 0))
```

Comment:
Also in general, is there a reason this is split from the metadata for idx > 0? Why not send all of them in one big loop?

Reply:
not as far as I'm aware, no; @kc5nra: do you know if there's a reason to split headers/metadata like this?

Reply:
This feedback thread is still active.

Reply:
changed in https://github.com/obsproject/obs-studio/pull/10500#commits-pushed-cf791ae